package.json

```diff
@@ -238,13 +257,19 @@
       {
         "language": "racket",
         "path": "./snippets/racket.json"
+      },
+      {
+        "language": "rhombus",
+        "path": "./snippets/rhombus.json"
       }
     ]
   },
   "scripts": {
     "vscode:prepublish": "npm run build-base -- --minify",
     "package": "vsce package -o magic-racket.vsix",
     "build-base": "esbuild ./src/extension.ts --bundle --outfile=out/main.js --external:vscode --format=cjs --platform=node --target=node16",
+    "build-syntax": "js-yaml syntaxes/src/racket.yaml > syntaxes/racket.tmLanguage.json && js-yaml syntaxes/src/rhombus.yaml > syntaxes/rhombus.tmLanguage.json",
```

Comment:
There weren't any instructions on how to update the syntax files. I figured it out through vscode's website, but it is nice to have some quick scripts for fast iteration on changes. Feel free to remove this if it isn't something you want.

Reply:
Seems reasonable (as long as we convince ourselves it gives good output), I think people had been just editing both files manually before.

You should add `js-yaml` as a dev dependency.

Reply:
I've added `js-yaml` to the dev dependencies.

Reply:
I've added a script to remove the comments as a postprocess step.